bin/colleagues.bash

```diff
@@ -29,19 +29,41 @@ function _Usage() {
     echo
 }
 
+function _PrintUserInfo() {
+    local _user="${1}"
+    local _format="${2}"
+    #echo "DEBUG: _user   = ${_user}."
+    #echo "DEBUG: _format = ${_format}."
+    IFS=':' read -a _user_info <<< "$(getent passwd ${_user} | cut -d ':' -s -f 1,5)"
```

Comment:
When IFS is redefined on the same line as the loop/code that parses the input, IFS is only redefined for that loop/code fragment. Only when you redefine IFS on a separate line before your loop/code fragment, you may have to reset it after your loop/code fragment to prevent unwanted side effects.